packages/manager/modules/pci/src/projects/project/notebooks/add/add.routing.js

```diff
@@ -11,6 +13,66 @@ export default /* @ngInject */ ($stateProvider) => {
         message,
         type,
       ) => goToNotebook(notebookInfo, message, type),
+
+      editors: /* @ngInject */ (projectId, NotebookService) =>
+        NotebookService.getEditors(projectId),
+
+      frameworks: /* @ngInject */ (projectId, NotebookService) =>
+        NotebookService.getFrameworks(projectId).then((frameworks) => {
+          return frameworks.map((f) => ({
+            ...f,
+            versions: f.versions.map((version) => ({ version })),
+          }));
+        }),
+
+      regions: /* @ngInject */ (projectId, NotebookService) =>
+        NotebookService.getRegions(projectId).then((regions) =>
+          regions.map((region) => ({
+            name: region,
+            hasEnoughQuota: () => true,
+          })),
+        ),
+
+      flavors: /* @ngInject */ (
```

Comment:
r u still waiting for the API? may be you can move this to separate class like notebook.mock.js and delete it when API is available.

Reply:
Completed